src/app/docs/wizard/wizard-docs.component.html

```diff
@@ -4,15 +4,11 @@
   pageTitle="Wizard"
 >
 
-<sky-docs-demo-page-summary>
-  <p>
-    Wizards guide users through pre-defined steps in a particular order to represent a series of related decisions in a user’s direct workflow. Wizards illustrate the step-wise nature of tasks and reinforce progress toward final goals as users complete steps. Wizards are most useful when users must perform steps in a particular order.
-  </p>
-
-  <sky-alert alertType="info">
```

Comment:
I know that you moved this down below. However, that will cause this to show up on all three progress indicator pages. Just curious as to why you wanted to move that out of the alert and into the properties themselves?

Reply:
I moved this based on Adam and Todd's guidance on the summary. They've said to keep the summary as short and pared down as possible and also to restrict implementation details to the Development tab.

This struck me as an implementation detail that could be restricted to the Development tab, but I don't feel strongly about it. This was just my best attempt to tighten it up and call out the detail with the property where it is implemented. I also wasn't worried about the fact that it shows up on all three pages because Steve said not to worry about that.

Do you think the change is potentially confusing or downplays the info too much? If so, we can switch back to the alert in the summary. What do you think?

Reply:
Personally - I think it is better served in the alert above. I'd rather keep it in the alert and we can change it later if Todd, Adam, or Steve feel strongly enough. I think that with what the content is that it could be confusing for it to be on all three pages. And I think that it could be missed and lead to confusion if anyone is going to the Wizard page looking for the old tabs style.

Reply:
I'll go with your recommendation. I was actually thinking it might be a good idea to downplay the old tabs style to avoid encouraging it, but I put the note back at the top and removed it from the property description. @